interfaces/Portal/src/app/pages/design/design.page.ts

```diff
@@ -1,7 +1,7 @@
 import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import Permission from 'src/app/auth/permission.enum';
-import { ProgramPhase } from 'src/app/models/program.model';
+import { ProgramTab } from 'src/app/models/program.model';
 
 @Component({
   selector: 'app-design',
```

Comment:
We could rename the page/component to `program-overview.page.ts` for example?

Or as a sub-page of `pages/program/*`, so `pages/program/overview/overview.page.ts`? (Like the `iframe-recipient`-page?)

Reply:
Changed to `program-overview*`